extensions/signal.sigmf-ext.md

```diff
@@ -0,0 +1,285 @@
+|`pm`|(analog) phase modulation|
+|`ssb`|(analog) single side-band|
+|`dsb`|(analog) dual side-band|
+|`vsb`|(analog) vestigal side-band|
+|`ask`|(analog) amplitude-shift keying|
+|`fsk`|(digital) frequency-shift keying|
+|`psk`|(digital) phase-shift keying|
+|`qam`|(digital) quadrature-amplitude modulation|
+|`ook`|(digital) on-off keying|
+|`cpm`|(digital) continuous phase modulation|
+|`msk`|(digital) minimum-shift keying|
+
+###### The `carrier_variant` Field
+
+The `carrier_variant` field can have the following values:
+
+|value|type|description|
+|----|----|-------|
+|`with_carrier`|(analog) with-carrier modulation|
+|`suppressed_carrier`|(analog) suppressed-carrier modulation|
+|`reduced_carrier`|(analog) reduced-carrier modulation|
+|`single_carrier`|(digital) single-carrier modulation|
+|`multi_carrier`|(digital) multi-carrier modulation|
+
+###### The `symbol_variant` Field
+
+The `symbol_variant` field can have the following values:
+
+|value|type|description|
+|----|----|-------|
+|`differential`|differential modulation|
+|`offset`|offset modulation (sometimes called 'staggered')|
+
+###### The `order` Field
+
+The `order` field has an unsigned integer value that describes the modulation order, which typically refers to the number of symbols or states in a digital modulation (e.g., QAM64 has 64 symbols, QPSK has 4 symbols).
+
+###### The `duplexing` Field
+
+The `duplexing` field can have the following values:
+
+|value|description|
+|----|-------|
+|`tdd`|time-division duplexing|
+|`fdd`|frequency-division duplexing|
+
+###### The `multiplexing` Field
+
+The `multiplexing` field can have the following values:
+
+|value|description|
+|----|-------|
+|`tdm`|time-division multiplexing|
+|`fdm`|frequency-division multiplexing|
+|`cdm`|code-division multiplexing|
+|`ofdm`|orthogonal frequency-division multiplexing|
+|`sdm`|space-division multiplexing|
+|`pdm`|polarization-division multiplexing|
+
+###### The `multiple_access` Field
+
+The `multiple_access` field can have the following values:
+
+|value|description|
+|----|-------|
+|`fdma`|frequency-division multiple access|
+|`ofdma`|orthogonal frequency-division multiple access|
+|`tdma`|time-division multiple access|
+|`cdma`|code-division multiple access|
+|`sdma`|space-division multiple access|
+|`pdma`|power-division multiple access|
+
+###### The `spreading` Field
+
+The `spreading` field can have the following values:
+
+|value|description|
+|----|-------|
+|`fhss`|frequency-hopping spread spectrum|
+|`thss`|time-hopping spread spectrum|
+|`dsss`|direct-sequence spread spectrum|
+|`css`|chirp spread spectrum|
+
+###### The `bandwidth` Field
+
+The `bandwidth` field has a float value that describes the channel bandwidth of the signal. Note that this is different from what is reported in the `core` namespace within an annotation, describing the occupied spectrum of a signal, which may or may not be comparable to the actual channel bandwidth of the communications system.
```

Comment:
Can we change this name to be more specific then? `channel_bw` or `channel_bandwidth`?

Reply:
Good point, and agreed with @jacobagilbert.